src/main/java/Duke/TaskList/TaskList.java

```diff
@@ -0,0 +1,150 @@
+package Duke.TaskList;
+
+import Duke.Task.Task;
+import Duke.Exception.*;
```

Comment:
Avoid wildcard imports.
```suggestion
import Duke.Exception.DukeException;
```